waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/FeatureCatalog.kt

```diff
@@ -66,5 +72,6 @@ object FeatureCatalog : ServiceFeatureCatalog {
         runtimeMockFeature,
         didWebRegistry,
         x5cAuthFeature,
```

Comment:
just wondering if there are use cases where only one of these features is used
would it make sense to combine them into single feature that would activate both the auth and the exchange endpoints?

Reply:
in principle, the x5c auth feature and the external signatures feature do not go together. However, the oid4vci endpoints will be added in this feature.